valhalla/baldr/traffictile.h

```diff
@@ -115,8 +116,9 @@ struct TrafficSpeed {
   }
 
   constexpr TrafficSpeed()
-      : overall_speed{0}, speed1{0}, speed2{0}, speed3{0}, breakpoint1{0}, breakpoint2{0},
-        congestion1{0}, congestion2{0}, congestion3{0}, has_incidents{0}, spare{0} {
+      : overall_speed{UNKNOWN_TRAFFIC_SPEED_RAW}, speed1{UNKNOWN_TRAFFIC_SPEED_RAW},
```

Comment:
this seems unfortunate. i suspect this will mean mean traffic tiles will be less compressor friendly

Reply:
Yeah - unfortunately, we're using `0` as "closed" :-/

Reply:
Remember, as long as `breakpoint1` is zero, the record is invalid.